### PR TITLE
Fixes #112, #113. Add API method and command to detect and remove rid…

### LINKED ISSUE #112: Add a command and API method to detect and remove or return ridges in an image
Using a ridge filter (built as the eigenvector of the hessian matrix), add the command and API method to detect ridge and return or remove them from the input image. Also add option to remove noise based on output connected component size.

### PR DESCRIPTION
**Pull request by versae**
Add API method and command to detect and remove ridges, specifically ridges representing text. Add tests and docs.

### CODE REVIEW

File: histonets/histonets.py [hunk 1]
```diff
@@ -239,3 +242,35 @@ def select_colors(image, colors, return_mask=False):
...
+    detection and removal.
+
+    A dilation kernel in pixels can be passed in, and the resulting mask can
+    also be returned instead when return_masl is set to True."""
+    gray_image = cv2.cvtColor(image, cv2.COLOR_RGB2GRAY)
+    # The value of sigma is calculated according to Steger's work:
+    # An Unbiased Detector of Curvilinear Structures,
+    # IEEE Transactions on Pattern Analysis and Machine Intelligence,
+    # Vol. 20, No. 2, Feb 1998
+    # http://ieeexplore.ieee.org/document/659930/
```
Comment (mejackreed) [inline histonets/histonets.py:262]: 👍

**Review by mejackreed — COMMENTED** [review]
👍  Looks great! Just one question (not-blocking). Mind responding?

File: requirements.txt [hunk 2]
```diff
@@ -9,3 +9,5 @@ Sphinx==1.4.8
 cryptography==1.4
 PyYAML==3.11
 codecov==2.0.5
+matplotlib==2.0.2
```
Comment (mejackreed) [inline requirements.txt]: Where is matplotlib being used now?

Comment (versae) [reply requirements.txt]: It's a dependency for the new version of `scikit-image`, but it should be installed by `scikit-image` itself. I'll check and remove it if necessary.

**Review by mejackreed — APPROVED** [review]
👍

**Pull request merged**